src/main/java/Duke.java

```diff
@@ -99,9 +103,12 @@ public static void main(String[] args) {
                     if (fromStart == 0 || toStart == 0) {
                         System.out.println("☹ OOPS!!! You must indicate ur event duration using /from and /to");
                     } else {
-                        String from = command.substring(fromStart, toStart-4);
+                        String from = command.substring(fromStart, toStart-5);
                         String to = command.substring(toStart);
-                        Event event = new Event(command.substring(0,fromStart-7), from, to);
+                        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+                        LocalDateTime formattedstartTime = LocalDateTime.parse(from, dateTimeFormatter);
+                        LocalDateTime formattedendTime= LocalDateTime.parse(to, dateTimeFormatter);
+                        Event event = new Event(command.substring(0,fromStart-7), formattedstartTime, formattedendTime);
```

Comment:
Hi, I'm your commenter and I would like to confirm whether the camelCase for these variable declarations has been fixed? Otherwise, great job on the clean code!